internal/uvm/create.go

```diff
@@ -108,8 +108,8 @@ type Options struct {
 	AdditionalHyperVConfig map[string]hcsschema.HvSocketServiceConfig
 
 	// The following options are for implicit vNUMA topology settings.
-	// MaxSizePerNode is the maximum size of memory per vNUMA node.
-	MaxSizePerNode uint64
+	// MaxMemorySizePerNumaNode is the maximum size of memory (in MiB) per vNUMA node.
```

Comment:
Is it really in MiB? HCS is in MiB?

Reply:
Yeah, from the examples I've seen and OS code (e.g., [[1]](https://microsoft.visualstudio.com/OS/_git/os.2020?path=/onecore/vm/compute/management/orchestration/shared/VmConfig/ConfigurationHelpers.cpp&version=GBofficial/main&line=972&lineEnd=977&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents), [[2]](https://microsoft.visualstudio.com/OS/_git/os.2020?path=/onecore/vm/common/mgmt/memory/VmMemorySettings.cpp&version=GBofficial/main&line=443&lineEnd=448&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)), its in MB